test-result-summary-client/src/utils/Utils.js

```diff
@@ -15,8 +15,8 @@ export const getInfoFromBuildName = (buildName) => {
 export const getGitDiffLinks = (before, after, buildName) => {
     const { jdkVersion } = getInfoFromBuildName(buildName);
     const diffLinks = {
-        "OpenJ9": "https://github.com/eclipse/openj9/compare/",
-        "OMR": "https://github.com/eclipse/omr/compare/",
+        "OpenJ9": "https://github.com/eclipse-openj9/openj9/compare/",
+        "OMR": "https://github.com/eclipse-openj9/omr/compare/",

```

Comment:
OMR repo is still https://github.com/eclipse/omr

Reply:
Corrected